src/Uno.UI/UI/Xaml/Style/Generic/DatePickerSelector.xaml

```diff
@@ -23,6 +23,8 @@
 				</ControlTemplate>
 			</Setter.Value>
 		</ios:Setter>
+        <ios:Setter Property="FlyoutPlacement"
```

Comment:
I think this is the only necessary change. DatePicker.cs and DatePicker.iOS.cs don't need to change.

Reply:
Not necessary but I did it to standardize with timepicker